packages/fxa-auth-server/fxa-oauth-server/scripts/gen_keys.js

```diff
@@ -12,75 +12,79 @@
  Will create these files
 
  ./config/key.json
+ ./config/newKey.json
  ./config/oldKey.json
 
+ This script is used for initializing keys in a dev environment.
  If these files already exist, this script will show an error message
- and exit. You must remove both keys if you want to generate a new
- keypair.
+ and exit. If you want to generate a new keypair, either use one of
+ the key-rotation scripts (e.g. 'prepare-new-signing-key.js') or just
+ delete the existing key file (hey, only it's a dev environment!).
  */
 
+//eslint-disable no-console
+
+// If you're running this script, you probably don't have keys created yet.
+// This bypasses config checks that would otherwise prevent us from running
+// without a properly-configured active key.
+process.env.FXA_OPENID_UNSAFELY_ALLOW_MISSING_ACTIVE_KEY = true;
+
 const fs = require('fs');
 const assert = require('assert');
-const crypto = require('crypto');
-const generateRSAKeypair = require('keypair');
-const JwTool = require('fxa-jwtool');
-
-const keyPath = './fxa-oauth-server/config/key.json';
-const oldKeyPath = './fxa-oauth-server/config/oldKey.json';
```

Comment:
So, I didn't like the way this was hard-coding paths, and duplicating values for our config files. I've changed it to look up the appropriate file paths from config, so that it'll run anywhere instead of only working for our dev configuration.

Supporting this is the reason that I've added a fun new `FXA_OPENID_UNSAFELY_ALLOW_MISSING_ACTIVE_KEY` config option. I think the code came up cleaner overall.